### PR TITLE
docs: add issue template

### PR DESCRIPTION
**Pull request by bashbunni**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Setup**
+Please complete the following information along with version numbers, if applicable.
+ - OS [e.g. Ubuntu, macOS]
+ - Shell [e.g. zsh, fish]
+ - Terminal Emulator [e.g. kitty, iterm]
+ - Terminal Multiplexer [e.g. tmux]
```
Comment (meowgorithm) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: I'd also add:

```
- Locale [e.g. en_US.UTF-8, zh_CN.UTF-8, etc.]
```

**Review by meowgorithm — CHANGES_REQUESTED** [review]
One small thing with locale, otherwise we're good to go.

I wonder if people need instructions on finding their locale? On unix systems I typically just call `locale`, but on Windows I believe it's a bit fussier. Given that perhaps we forgo it for now unless we start hear it's an issue, if you agree.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (bashbunni) [reply .github/ISSUE_TEMPLATE/bug_report.md:18]: Agreed!

**Comment by bashbunni** [comment]
@meowgorithm added the locale and added the locale to the PRs in Glow and Lip Gloss as well for the issue templates over there. 

https://github.com/charmbracelet/glow/pull/486
https://github.com/charmbracelet/lipgloss/pull/193

**Pull request merged**